docker/Dockerfile

```diff
@@ -0,0 +1,5 @@
+#Import Ruby base image(change version as needed)
+FROM ruby:2.7-rc
```

Comment:
Looks like `2.7-rc` is not an [official release](https://www.ruby-lang.org/en/downloads/releases/). I think we should specify the latest official release instead of a release candidate.